monitoring/procfs_net.go

```diff
@@ -63,8 +63,11 @@ func (r portCollector) sockets(fetchSockets ...socketGetterFunc) (ret []process,
 
 	for _, socket := range sockets {
 		proc, err := r.findProcessByInode(socket.inode())
-		if err != nil {
-			log.Warn(err.Error())
+		if err != nil && socket.inode() != "0" && !trace.IsNotFound(err) {
```

Comment:
Could you either add a comment explaining what this condition checks for, or move it into a helper method with descriptive name?